docs/grizli/install.rst

```diff
@@ -64,16 +188,15 @@ Download and set up ``grizli``
 .. code-block:: bash
 
    pip install --editable .
+   
+Or to install the optional dependencies:
 
-.. note::
-   The `--editable` flag builds the Cython extensions needed for `pytest`.
+.. code-block:: bash
 
-- Install ``hstcal``, a dependency for the WFC3/IR pipeline ``calwf3``, that doesn't
-  install with ``pip``:
+   pip install --editable ".[jwst,test]"
```

Comment:
`-e` instead of `--editable` is probably a good shortcut that people should know and use.